components/feature/recents/src/main/java/mozilla/components/feature/recents/db/RecentAppsDatabase.kt

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.recents.db
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+/**
+ * Internal database for storing apps and their scores that determine if they are most recently used.
+ */
+@Database(entities = [RecentAppEntity::class], version = 1)
+internal abstract class RecentAppsDatabase : RoomDatabase() {
```

Comment:
My only nit is to rename the database to `RecentsDatabase` if we decide to add other things in here as well later on we then have one database for it and multiple tables for each.

Reply:
Renamed as Sebastian proposed :)